validator/validator-main.protoascii

```diff
@@ -4139,6 +4139,13 @@ attr_lists: {
         ")(\\s|$)"
   }
 }
+attr_lists: {
```

Comment:
Nit: this isn't a list, this is a single attribute. Why not put the attribute directly on the tagspecs that need it?

Reply:
Glad to make the change. I felt like it was a good way to self-document and make it easy to add future attributes from amp-form to `input` tags.

Reply:
If there are planned additional attributes for this list then by all means keep this structure. If we don't have any planned then I'd prefer applying directly to the tagspec.